src/System.Security.Permissions/src/System/Security/PermissionSet.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace System.Security
+{
+    public partial class PermissionSet : System.Collections.ICollection, System.Collections.IEnumerable, /* System.Runtime.Serialization.IDeserializationCallback, */ System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        public PermissionSet(System.Security.Permissions.PermissionState state) { }
+        public PermissionSet(System.Security.PermissionSet permSet) { }
+        public virtual int Count { get { return default(int); } }
+        public virtual bool IsReadOnly { get { return default(bool); } }
+        public virtual bool IsSynchronized { get { return default(bool); } }
+        public virtual object SyncRoot { get { return default(object); } }
+        public System.Security.IPermission AddPermission(System.Security.IPermission perm) { return default(System.Security.IPermission); }
+        public void Assert() { }
+        public bool ContainsNonCodeAccessPermissions() { return default(bool); }
+        public static byte[] ConvertPermissionSet(string inFormat, byte[] inData, string outFormat) { return default(byte[]); }
+        public virtual System.Security.PermissionSet Copy() { return default(System.Security.PermissionSet); }
+        public virtual void CopyTo(System.Array array, int index) { }
+        public void Demand() { }
+        public void Deny() { }
+        public override bool Equals(object obj) { return default(bool); }
+        //    public virtual void FromXml(System.Security.SecurityElement et) { }
+        public System.Collections.IEnumerator GetEnumerator() { return default(System.Collections.IEnumerator); }
+        public override int GetHashCode() { return default(int); }
```

Comment:
My previous comments about GetHashCode and Equals, as with ToString, should generally apply to all of these overrides throughout the PR.  Unless we're going to provide a real implementation, or at least something more than returning default(T), we should either just remove the overrides, or if we need them to be 100% match to the contract, change the overrides to call the base method.